Automattic/wp-calypso
# Sensei Onboarding - Introduce SenseiStepProgress component.

#### Proposed Changes

* Adds a `SenseiStepProgress` component that could be reused to show processing/progressing message to the user.

#### Testing Instructions



* Go through the Sensei Tailored Flow and confirm that the screens after choosing a Plan and after checkout looks like the following.
<img width="1707" alt="Screen Shot 2022-11-02 at 15 21 13" src="https://user-images.githubusercontent.com/2578542/199477318-8ecd2166-8861-4635-b166-0108bbe8f04e.png">

#### Pre-merge Checklist






Related to [#Automattic/sensei-pro/1870](https://github.com/Automattic/sensei-pro/issues/1870)
